### DISCUSSION TITLE
Not accepting my address -- why?

### DISCUSSION THREAD

**Discussion by JWegener** [davetapley/helium-tax#55] [Q&A] [answered]
I'm trying it with 
14QHAZzxqvGiivAB4QbuN3kibTsFMn6tuHunU1x65CNAKPGdzbz
and getting an error but https://www.fairspot.host/hnt-export-mining-tax seems to work fine.

**Comment by JWegener** [comment]
Does it only do Miner earnings -- not validator earnings and all transactions in and out of a helium wallet?

**Comment by davetapley** [comment, marked as answer]
No validator support yet, but you're not the first to ask, see #40
